infrastructure_files/getting-started-with-zitadel.sh

```diff
@@ -430,6 +430,12 @@ initEnvironment() {
   TURN_MAX_PORT=65535
   TURN_EXTERNAL_IP_CONFIG=$(get_turn_external_ip)
 
+  if [ ! "$ZITADEL_DATABASE" ]; then
```

Comment:
Hello @r0b2g1t this was resolved with the change we pushed in https://github.com/netbirdio/netbird/pull/2194

Reply:
I saw the merges, but with the set of env variable ZITADEL_DATABASE to postgres it was not able to for to us it, which was discussed in my last PR. with checking the the variable explicit of the content postgres you are more flexible for further changes and the use of other databases and there differentiation.

Reply:
The effect is similar since we want to default to Postgres; we can initialize the ZITADEL_DATABASE if it doesn't match the cockroach with Postgres, and, in the future, do the same when there are other engines supported. So we would just add another condition before the final else for each new engine.

Reply:
Yes, thats true it works also in this way. Thanks for your explanation. :-)